src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -820,8 +823,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             // the first paint will be ignored!
             dxEngine->SetWarningCallback(std::bind(&TermControl::_RendererWarning, this, std::placeholders::_1));
 
-            dxEngine->SetRetroTerminalEffect(_settings.RetroTerminalEffect());

```

Comment:
why did these get nuked?

Reply:
oh, they're in the appearance setup aren't they!

Reply:
Yep, they're in the `_UpdateAppearanceFromUIThreadUnderLock` call below (really should have been nuked in the original PR tbh)